# Regression with Keras

Import the libraries and importing keras model amd layers

In [1]:
import pandas as pd
import numpy as np
import keras
from keras.models import Sequential
from keras.layers import Dense
from sklearn.model_selection import train_test_split

In [2]:
data=pd.read_csv("concrete_data.csv")
data.head()

Cement  Blast Furnace Slag  Fly Ash  Water  Superplasticizer  \
0   540.0                 0.0      0.0  162.0               2.5   
1   540.0                 0.0      0.0  162.0               2.5   
2   332.5               142.5      0.0  228.0               0.0   
3   332.5               142.5      0.0  228.0               0.0   
4   198.6               132.4      0.0  192.0               0.0   

   Coarse Aggregate  Fine Aggregate  Age  Strength  
0            1040.0           676.0   28     79.99  
1            1055.0           676.0   28     61.89  
2             932.0           594.0  270     40.27  
3             932.0           594.0  365     41.05  
4             978.4           825.5  360     44.30

In [3]:
data.shape

(1030, 9)

In [4]:
data.describe()

Cement  Blast Furnace Slag      Fly Ash        Water  \
count  1030.000000         1030.000000  1030.000000  1030.000000   
mean    281.167864           73.895825    54.188350   181.567282   
std     104.506364           86.279342    63.997004    21.354219   
min     102.000000            0.000000     0.000000   121.800000   
25%     192.375000            0.000000     0.000000   164.900000   
50%     272.900000           22.000000     0.000000   185.000000   
75%     350.000000          142.950000   118.300000   192.000000   
max     540.000000          359.400000   200.100000   247.000000   

       Superplasticizer  Coarse Aggregate  Fine Aggregate          Age  \
count       1030.000000       1030.000000     1030.000000  1030.000000   
mean           6.204660        972.918932      773.580485    45.662136   
std            5.973841         77.753954       80.175980    63.169912   
min            0.000000        801.000000      594.000000     1.000000   
25%            0.000000        932.000000      730.950000     7.000000   
50%            6.400000        968.000000      779.500000    28.000000   
75%           10.200000       1029.400000      824.000000    56.000000   
max           32.200000       1145.000000      992.600000   365.000000   

          Strength  
count  1030.000000  
mean     35.817961  
std      16.705742  
min       2.330000  
25%      23.710000  
50%      34.445000  
75%      46.135000  
max      82.600000

So the first concrete sample has 540 cubic meter of cement, 0 cubic meter of blast furnace slag, 0 cubic meter of fly ash, 162 cubic meter of water, 2.5 cubic meter of superplaticizer, 1040 cubic meter of coarse aggregate, 676 cubic meter of fine aggregate. Such a concrete mix which is 28 days old, has a compressive strength of 79.99 MPa. 


In [5]:
data.isnull().sum()

Cement                0
Blast Furnace Slag    0
Fly Ash               0
Water                 0
Superplasticizer      0
Coarse Aggregate      0
Fine Aggregate        0
Age                   0
Strength              0
dtype: int64

#### Split data into predictors and target

In [6]:
predictor=data.drop(columns='Strength')
target=data['Strength']

In [7]:
# predictor_norm=(predictor-predictor.mean())/predictor.std()
# predictor_norm.head()

In [8]:
X_train,X_test,Y_train,Y_test=train_test_split(predictor,target,test_size=0.3)

Let's do a quick sanity check of the predictors and the target dataframes.

#### Normalizing Data

In [10]:
n_cols=predictor.shape[1] 

## Build a Neural Network

In [11]:
def regression_model():
    
    # create model
    model=Sequential()
    model.add(Dense(10,activation='relu',input_shape=(n_cols,)))
    model.add(Dense(1))
    
    model.compile(optimizer='adam',loss='mean_squared_error')
    return model

The above function create a model that has three hidden layers, each of 5 hidden units.

## Train and Test the Network  

In [12]:
model=regression_model()

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


In [14]:
model.fit(X_train,Y_train,epochs=50)

Epoch 1/50
23/23 ━━━━━━━━━━━━━━━━━━━━ 0s 2ms/step - loss: 126.5491 
Epoch 2/50
23/23 ━━━━━━━━━━━━━━━━━━━━ 0s 2ms/step - loss: 117.8388
Epoch 3/50
23/23 ━━━━━━━━━━━━━━━━━━━━ 0s 2ms/step - loss: 123.5921 
Epoch 4/50
23/23 ━━━━━━━━━━━━━━━━━━━━ 0s 1ms/step - loss: 121.9281 
Epoch 5/50
23/23 ━━━━━━━━━━━━━━━━━━━━ 0s 2ms/step - loss: 116.0826 
Epoch 6/50
23/23 ━━━━━━━━━━━━━━━━━━━━ 0s 2ms/step - loss: 119.2868 
Epoch 7/50
23/23 ━━━━━━━━━━━━━━━━━━━━ 0s 2ms/step - loss: 112.7817 
Epoch 8/50
23/23 ━━━━━━━━━━━━━━━━━━━━ 0s 2ms/step - loss: 116.4172 
Epoch 9/50
23/23 ━━━━━━━━━━━━━━━━━━━━ 0s 2ms/step - loss: 108.4794
Epoch 10/50
23/23 ━━━━━━━━━━━━━━━━━━━━ 0s 2ms/step - loss: 123.7809 
Epoch 11/50
23/23 ━━━━━━━━━━━━━━━━━━━━ 0s 2ms/step - loss: 112.1993
Epoch 12/50
23/23 ━━━━━━━━━━━━━━━━━━━━ 0s 2ms/step - loss: 123.3384 
Epoch 13/50
23/23 ━━━━━━━━━━━━━━━━━━━━ 0s 2ms/step - loss: 110.4905
Epoch 14/50
23/23 ━━━━━━━━━━━━━━━━━━━━ 0s 1ms/step - loss: 120.3307 
Epoch 15/50
23/23 ━━━━━━━━━━━━━━━━━━━━ 0s 2ms/s

In [15]:
predictions=model.predict(X_test)

10/10 ━━━━━━━━━━━━━━━━━━━━ 0s 5ms/step 


In [16]:
predictions
# model.compile(optimizer='adam',loss='mean_squared_error')

array([[16.854254],
       [51.30486 ],
       [31.639975],
       [36.91759 ],
       [37.00411 ],
       [37.23128 ],
       [65.53797 ],
       [32.68047 ],
       [34.007893],
       [35.94176 ],
       [36.081287],
       [51.821888],
       [28.247198],
       [12.838719],
       [24.786337],
       [42.079884],
       [38.78272 ],
       [30.961966],
       [52.263004],
       [19.058554],
       [27.2732  ],
       [49.671116],
       [58.07085 ],
       [36.02666 ],
       [48.028522],
       [33.512775],
       [26.52239 ],
       [25.63622 ],
       [16.849142],
       [36.197056],
       [28.944723],
       [35.846836],
       [62.40221 ],
       [30.264288],
       [39.404392],
       [30.434896],
       [53.579792],
       [49.21938 ],
       [31.008718],
       [25.102758],
       [53.503315],
       [25.48223 ],
       [49.17879 ],
       [24.493048],
       [30.527761],
       [59.173542],
       [50.59212 ],
       [25.176474],
       [40.155598],
       [63.535892],


In [20]:
n_cols = predictor.shape[1] # number of predictors
mean_squared_errors = []
from sklearn.metrics import mean_squared_error

for i in range(50):
    x_train, x_test, y_train, y_test = train_test_split(predictor, target, test_size=0.3)
    model = regression_model()
    model.fit(x_train, y_train, validation_data=(x_test, y_test), epochs=50, verbose=2)
    
    predictions=model.predict(X_test)
    mse = mean_squared_error(Y_test, predictions)
    mean_squared_errors.append(mse)
    



Epoch 1/50


C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 50ms/step - loss: 339054.3438 - val_loss: 223099.8906
Epoch 2/50
23/23 - 0s - 5ms/step - loss: 150230.0156 - val_loss: 88359.3828
Epoch 3/50
23/23 - 0s - 5ms/step - loss: 51860.8945 - val_loss: 22646.9141
Epoch 4/50
23/23 - 0s - 5ms/step - loss: 11076.2305 - val_loss: 4846.7817
Epoch 5/50
23/23 - 0s - 5ms/step - loss: 3797.7258 - val_loss: 3609.2131
Epoch 6/50
23/23 - 0s - 5ms/step - loss: 3351.3218 - val_loss: 3366.3000
Epoch 7/50
23/23 - 0s - 4ms/step - loss: 3115.3877 - val_loss: 3133.8833
Epoch 8/50
23/23 - 0s - 5ms/step - loss: 2896.5564 - val_loss: 2911.8652
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 2689.5146 - val_loss: 2696.7683
Epoch 10/50
23/23 - 0s - 5ms/step - loss: 2495.3926 - val_loss: 2491.0288
Epoch 11/50
23/23 - 0s - 5ms/step - loss: 2304.8164 - val_loss: 2309.3550
Epoch 12/50
23/23 - 0s - 6ms/step - loss: 2135.7114 - val_loss: 2131.6709
Epoch 13/50
23/23 - 0s - 5ms/step - loss: 1978.9510 - val_loss: 1974.5206
Epoch 14/50
23/23 - 0s - 5ms/step - loss: 1833.

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 48ms/step - loss: 159014.8594 - val_loss: 89327.2344
Epoch 2/50
23/23 - 0s - 5ms/step - loss: 52388.1406 - val_loss: 23297.1582
Epoch 3/50
23/23 - 0s - 4ms/step - loss: 12144.9277 - val_loss: 5531.3726
Epoch 4/50
23/23 - 0s - 5ms/step - loss: 4382.1387 - val_loss: 3874.4817
Epoch 5/50
23/23 - 0s - 5ms/step - loss: 3765.5430 - val_loss: 3403.6609
Epoch 6/50
23/23 - 0s - 5ms/step - loss: 3293.3870 - val_loss: 2999.7896
Epoch 7/50
23/23 - 0s - 4ms/step - loss: 2900.5510 - val_loss: 2662.1931
Epoch 8/50
23/23 - 0s - 5ms/step - loss: 2606.3884 - val_loss: 2404.9961
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 2392.2866 - val_loss: 2197.4617
Epoch 10/50
23/23 - 0s - 4ms/step - loss: 2197.7222 - val_loss: 2030.9963
Epoch 11/50
23/23 - 0s - 4ms/step - loss: 2033.1652 - val_loss: 1871.5267
Epoch 12/50
23/23 - 0s - 4ms/step - loss: 1873.5187 - val_loss: 1718.1223
Epoch 13/50
23/23 - 0s - 5ms/step - loss: 1719.9696 - val_loss: 1581.6270
Epoch 14/50
23/23 - 0s - 4ms/step - loss: 1581.5178

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 48ms/step - loss: 1954.5486 - val_loss: 1176.0656
Epoch 2/50
23/23 - 0s - 5ms/step - loss: 961.9493 - val_loss: 666.9708
Epoch 3/50
23/23 - 0s - 5ms/step - loss: 683.3569 - val_loss: 528.0212
Epoch 4/50
23/23 - 0s - 7ms/step - loss: 569.8394 - val_loss: 457.8777
Epoch 5/50
23/23 - 0s - 5ms/step - loss: 499.3544 - val_loss: 388.3990
Epoch 6/50
23/23 - 0s - 5ms/step - loss: 425.1645 - val_loss: 342.9727
Epoch 7/50
23/23 - 0s - 5ms/step - loss: 352.7809 - val_loss: 279.0295
Epoch 8/50
23/23 - 0s - 5ms/step - loss: 289.1797 - val_loss: 232.1883
Epoch 9/50
23/23 - 0s - 5ms/step - loss: 231.6572 - val_loss: 199.0327
Epoch 10/50
23/23 - 0s - 5ms/step - loss: 199.8461 - val_loss: 176.3147
Epoch 11/50
23/23 - 0s - 5ms/step - loss: 174.6524 - val_loss: 177.2166
Epoch 12/50
23/23 - 0s - 5ms/step - loss: 158.5839 - val_loss: 149.8104
Epoch 13/50
23/23 - 0s - 5ms/step - loss: 144.2032 - val_loss: 148.0760
Epoch 14/50
23/23 - 0s - 5ms/step - loss: 136.0019 - val_loss: 139.5614
Epoch 15/

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 45ms/step - loss: 40962.7461 - val_loss: 13944.1992
Epoch 2/50
23/23 - 0s - 4ms/step - loss: 8057.1167 - val_loss: 5805.0151
Epoch 3/50
23/23 - 0s - 4ms/step - loss: 5139.8774 - val_loss: 4876.2627
Epoch 4/50
23/23 - 0s - 4ms/step - loss: 4148.7563 - val_loss: 3967.6707
Epoch 5/50
23/23 - 0s - 4ms/step - loss: 3397.2126 - val_loss: 3350.4231
Epoch 6/50
23/23 - 0s - 4ms/step - loss: 2829.0969 - val_loss: 2873.2185
Epoch 7/50
23/23 - 0s - 4ms/step - loss: 2407.7764 - val_loss: 2491.1665
Epoch 8/50
23/23 - 0s - 4ms/step - loss: 2084.1865 - val_loss: 2183.8083
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 1823.1746 - val_loss: 1933.0498
Epoch 10/50
23/23 - 0s - 4ms/step - loss: 1609.7998 - val_loss: 1714.8535
Epoch 11/50
23/23 - 0s - 5ms/step - loss: 1429.1598 - val_loss: 1537.3033
Epoch 12/50
23/23 - 0s - 4ms/step - loss: 1276.7935 - val_loss: 1373.2811
Epoch 13/50
23/23 - 0s - 4ms/step - loss: 1142.4263 - val_loss: 1238.6737
Epoch 14/50
23/23 - 0s - 5ms/step - loss: 1035.6726 - v

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 45ms/step - loss: 15263.5703 - val_loss: 4514.7251
Epoch 2/50
23/23 - 0s - 4ms/step - loss: 1809.7668 - val_loss: 721.1983
Epoch 3/50
23/23 - 0s - 5ms/step - loss: 744.0074 - val_loss: 670.2971
Epoch 4/50
23/23 - 0s - 4ms/step - loss: 663.4901 - val_loss: 580.0418
Epoch 5/50
23/23 - 0s - 4ms/step - loss: 593.0322 - val_loss: 521.5919
Epoch 6/50
23/23 - 0s - 4ms/step - loss: 538.3452 - val_loss: 471.1484
Epoch 7/50
23/23 - 0s - 4ms/step - loss: 492.6925 - val_loss: 428.5774
Epoch 8/50
23/23 - 0s - 4ms/step - loss: 456.3584 - val_loss: 391.9727
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 422.7233 - val_loss: 363.4624
Epoch 10/50
23/23 - 0s - 10ms/step - loss: 394.4507 - val_loss: 338.4536
Epoch 11/50
23/23 - 0s - 4ms/step - loss: 370.9595 - val_loss: 317.1896
Epoch 12/50
23/23 - 0s - 4ms/step - loss: 349.6867 - val_loss: 297.9108
Epoch 13/50
23/23 - 0s - 4ms/step - loss: 332.4442 - val_loss: 281.6857
Epoch 14/50
23/23 - 0s - 4ms/step - loss: 315.1243 - val_loss: 268.0897
Epoch 

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 53ms/step - loss: 179587.4375 - val_loss: 115424.1562
Epoch 2/50
23/23 - 0s - 4ms/step - loss: 75556.2188 - val_loss: 41171.1562
Epoch 3/50
23/23 - 0s - 4ms/step - loss: 23172.5566 - val_loss: 10377.1816
Epoch 4/50
23/23 - 0s - 4ms/step - loss: 5311.7671 - val_loss: 2560.7756
Epoch 5/50
23/23 - 0s - 5ms/step - loss: 1736.9060 - val_loss: 1541.6517
Epoch 6/50
23/23 - 0s - 6ms/step - loss: 1396.9219 - val_loss: 1430.9631
Epoch 7/50
23/23 - 0s - 6ms/step - loss: 1332.9124 - val_loss: 1370.1677
Epoch 8/50
23/23 - 0s - 5ms/step - loss: 1278.5311 - val_loss: 1305.7466
Epoch 9/50
23/23 - 0s - 5ms/step - loss: 1223.3539 - val_loss: 1250.0594
Epoch 10/50
23/23 - 0s - 4ms/step - loss: 1172.8683 - val_loss: 1189.4230
Epoch 11/50
23/23 - 0s - 5ms/step - loss: 1122.4573 - val_loss: 1136.1196
Epoch 12/50
23/23 - 0s - 5ms/step - loss: 1075.9908 - val_loss: 1085.0159
Epoch 13/50
23/23 - 0s - 5ms/step - loss: 1030.8271 - val_loss: 1032.0887
Epoch 14/50
23/23 - 0s - 5ms/step - loss: 989.115

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 60ms/step - loss: 163937.8281 - val_loss: 104348.6484
Epoch 2/50
23/23 - 0s - 5ms/step - loss: 65590.9844 - val_loss: 37340.2734
Epoch 3/50
23/23 - 0s - 4ms/step - loss: 21166.3262 - val_loss: 11459.3164
Epoch 4/50
23/23 - 0s - 5ms/step - loss: 6484.3730 - val_loss: 4212.7441
Epoch 5/50
23/23 - 0s - 4ms/step - loss: 3264.5881 - val_loss: 2898.5935
Epoch 6/50
23/23 - 0s - 4ms/step - loss: 2800.6052 - val_loss: 2676.7271
Epoch 7/50
23/23 - 0s - 4ms/step - loss: 2686.6694 - val_loss: 2580.1733
Epoch 8/50
23/23 - 0s - 5ms/step - loss: 2585.6699 - val_loss: 2481.2920
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 2483.4395 - val_loss: 2391.3713
Epoch 10/50
23/23 - 0s - 4ms/step - loss: 2379.0029 - val_loss: 2290.5312
Epoch 11/50
23/23 - 0s - 4ms/step - loss: 2276.6504 - val_loss: 2191.4871
Epoch 12/50
23/23 - 0s - 5ms/step - loss: 2176.7036 - val_loss: 2101.4233
Epoch 13/50
23/23 - 0s - 4ms/step - loss: 2067.0667 - val_loss: 1984.5139
Epoch 14/50
23/23 - 0s - 4ms/step - loss: 1963.22

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 46ms/step - loss: 494051.7812 - val_loss: 337362.5000
Epoch 2/50
23/23 - 0s - 5ms/step - loss: 247046.3750 - val_loss: 159860.5781
Epoch 3/50
23/23 - 0s - 4ms/step - loss: 115598.4141 - val_loss: 72280.4531
Epoch 4/50
23/23 - 0s - 4ms/step - loss: 52098.7812 - val_loss: 30770.9551
Epoch 5/50
23/23 - 0s - 4ms/step - loss: 22443.7422 - val_loss: 12832.1367
Epoch 6/50
23/23 - 0s - 4ms/step - loss: 9927.0088 - val_loss: 5743.1655
Epoch 7/50
23/23 - 0s - 4ms/step - loss: 5125.5312 - val_loss: 3402.3850
Epoch 8/50
23/23 - 0s - 4ms/step - loss: 3586.4348 - val_loss: 2709.3525
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 3065.5056 - val_loss: 2505.8525
Epoch 10/50
23/23 - 0s - 4ms/step - loss: 2859.6687 - val_loss: 2391.4473
Epoch 11/50
23/23 - 0s - 5ms/step - loss: 2709.8945 - val_loss: 2287.2341
Epoch 12/50
23/23 - 0s - 4ms/step - loss: 2582.3469 - val_loss: 2180.5693
Epoch 13/50
23/23 - 0s - 4ms/step - loss: 2458.1477 - val_loss: 2076.0420
Epoch 14/50
23/23 - 0s - 4ms/step - loss: 

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 60ms/step - loss: 230984.5469 - val_loss: 187707.7969
Epoch 2/50
23/23 - 0s - 4ms/step - loss: 159876.8438 - val_loss: 132093.6719
Epoch 3/50
23/23 - 0s - 4ms/step - loss: 114755.7422 - val_loss: 96633.1250
Epoch 4/50
23/23 - 0s - 4ms/step - loss: 85529.4375 - val_loss: 73111.4922
Epoch 5/50
23/23 - 0s - 4ms/step - loss: 65574.1094 - val_loss: 56575.8945
Epoch 6/50
23/23 - 0s - 4ms/step - loss: 51257.2930 - val_loss: 44601.2656
Epoch 7/50
23/23 - 0s - 4ms/step - loss: 40812.7422 - val_loss: 35831.4570
Epoch 8/50
23/23 - 0s - 4ms/step - loss: 33023.0430 - val_loss: 29189.4766
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 27057.5977 - val_loss: 24067.5820
Epoch 10/50
23/23 - 0s - 4ms/step - loss: 22382.7207 - val_loss: 20013.2539
Epoch 11/50
23/23 - 0s - 4ms/step - loss: 18644.7773 - val_loss: 16731.8887
Epoch 12/50
23/23 - 0s - 4ms/step - loss: 15622.0518 - val_loss: 14035.0225
Epoch 13/50
23/23 - 0s - 4ms/step - loss: 13129.9414 - val_loss: 11794.9170
Epoch 14/50
23/23 - 0s - 4

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 58ms/step - loss: 320169.8438 - val_loss: 247330.2031
Epoch 2/50
23/23 - 0s - 7ms/step - loss: 190075.8438 - val_loss: 135932.7500
Epoch 3/50
23/23 - 0s - 16ms/step - loss: 93466.2734 - val_loss: 52071.2227
Epoch 4/50
23/23 - 0s - 7ms/step - loss: 25901.6367 - val_loss: 5890.1309
Epoch 5/50
23/23 - 0s - 12ms/step - loss: 2328.3215 - val_loss: 1174.0605
Epoch 6/50
23/23 - 0s - 7ms/step - loss: 1113.2721 - val_loss: 1021.8804
Epoch 7/50
23/23 - 0s - 8ms/step - loss: 948.2861 - val_loss: 916.4674
Epoch 8/50
23/23 - 0s - 8ms/step - loss: 885.6712 - val_loss: 857.1883
Epoch 9/50
23/23 - 0s - 8ms/step - loss: 833.0502 - val_loss: 803.8219
Epoch 10/50
23/23 - 0s - 7ms/step - loss: 789.9199 - val_loss: 755.4398
Epoch 11/50
23/23 - 0s - 6ms/step - loss: 743.3799 - val_loss: 714.5798
Epoch 12/50
23/23 - 0s - 5ms/step - loss: 708.9221 - val_loss: 674.2945
Epoch 13/50
23/23 - 0s - 4ms/step - loss: 673.8255 - val_loss: 639.5848
Epoch 14/50
23/23 - 0s - 4ms/step - loss: 643.8416 - val_l

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 46ms/step - loss: 146650.5781 - val_loss: 78516.7969
Epoch 2/50
23/23 - 0s - 4ms/step - loss: 41702.7305 - val_loss: 18814.1445
Epoch 3/50
23/23 - 0s - 5ms/step - loss: 11732.4219 - val_loss: 7258.0430
Epoch 4/50
23/23 - 0s - 5ms/step - loss: 8157.5444 - val_loss: 6031.2837
Epoch 5/50
23/23 - 0s - 4ms/step - loss: 7609.7441 - val_loss: 5708.8359
Epoch 6/50
23/23 - 0s - 4ms/step - loss: 7120.8989 - val_loss: 5407.5674
Epoch 7/50
23/23 - 0s - 4ms/step - loss: 6646.7939 - val_loss: 5081.6489
Epoch 8/50
23/23 - 0s - 4ms/step - loss: 6180.1138 - val_loss: 4718.5044
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 5754.2402 - val_loss: 4419.3359
Epoch 10/50
23/23 - 0s - 4ms/step - loss: 5329.2432 - val_loss: 4111.2915
Epoch 11/50
23/23 - 0s - 4ms/step - loss: 4933.8584 - val_loss: 3840.2253
Epoch 12/50
23/23 - 0s - 5ms/step - loss: 4553.8774 - val_loss: 3529.3127
Epoch 13/50
23/23 - 0s - 4ms/step - loss: 4191.8394 - val_loss: 3298.6331
Epoch 14/50
23/23 - 0s - 4ms/step - loss: 3865.5212

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 46ms/step - loss: 173714.2188 - val_loss: 125716.2109
Epoch 2/50
23/23 - 0s - 5ms/step - loss: 91979.8047 - val_loss: 58353.8438
Epoch 3/50
23/23 - 0s - 4ms/step - loss: 36567.5781 - val_loss: 17823.0332
Epoch 4/50
23/23 - 0s - 4ms/step - loss: 11783.2744 - val_loss: 8130.9941
Epoch 5/50
23/23 - 0s - 4ms/step - loss: 7882.8184 - val_loss: 7265.1343
Epoch 6/50
23/23 - 0s - 4ms/step - loss: 7098.5396 - val_loss: 6571.3911
Epoch 7/50
23/23 - 0s - 4ms/step - loss: 6292.9668 - val_loss: 5721.6777
Epoch 8/50
23/23 - 0s - 4ms/step - loss: 5322.9780 - val_loss: 4749.2344
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 4361.9448 - val_loss: 3951.3276
Epoch 10/50
23/23 - 0s - 4ms/step - loss: 3574.9326 - val_loss: 3243.3313
Epoch 11/50
23/23 - 0s - 4ms/step - loss: 3004.6392 - val_loss: 2776.1865
Epoch 12/50
23/23 - 0s - 4ms/step - loss: 2577.2234 - val_loss: 2419.2434
Epoch 13/50
23/23 - 0s - 4ms/step - loss: 2273.9800 - val_loss: 2141.6453
Epoch 14/50
23/23 - 0s - 4ms/step - loss: 2006.3

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 50ms/step - loss: 2871.5671 - val_loss: 2213.8738
Epoch 2/50
23/23 - 0s - 4ms/step - loss: 1748.8103 - val_loss: 1503.1766
Epoch 3/50
23/23 - 0s - 4ms/step - loss: 1197.5814 - val_loss: 980.5536
Epoch 4/50
23/23 - 0s - 5ms/step - loss: 818.8049 - val_loss: 708.2031
Epoch 5/50
23/23 - 0s - 5ms/step - loss: 597.4703 - val_loss: 552.2349
Epoch 6/50
23/23 - 0s - 4ms/step - loss: 474.1034 - val_loss: 449.5529
Epoch 7/50
23/23 - 0s - 4ms/step - loss: 378.7325 - val_loss: 381.6292
Epoch 8/50
23/23 - 0s - 4ms/step - loss: 326.7049 - val_loss: 336.2834
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 291.7464 - val_loss: 314.0311
Epoch 10/50
23/23 - 0s - 5ms/step - loss: 266.4421 - val_loss: 282.2195
Epoch 11/50
23/23 - 0s - 4ms/step - loss: 242.8529 - val_loss: 266.5163
Epoch 12/50
23/23 - 0s - 5ms/step - loss: 225.9054 - val_loss: 248.2580
Epoch 13/50
23/23 - 0s - 4ms/step - loss: 219.4483 - val_loss: 234.4393
Epoch 14/50
23/23 - 0s - 4ms/step - loss: 203.7189 - val_loss: 218.5942
Epoch 

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 46ms/step - loss: 147553.5781 - val_loss: 108797.3047
Epoch 2/50
23/23 - 0s - 4ms/step - loss: 80533.7422 - val_loss: 59357.3711
Epoch 3/50
23/23 - 0s - 4ms/step - loss: 43132.9453 - val_loss: 32177.0605
Epoch 4/50
23/23 - 0s - 4ms/step - loss: 22744.2129 - val_loss: 17263.6465
Epoch 5/50
23/23 - 0s - 4ms/step - loss: 11883.0859 - val_loss: 9568.1582
Epoch 6/50
23/23 - 0s - 4ms/step - loss: 6496.0117 - val_loss: 5813.2148
Epoch 7/50
23/23 - 0s - 4ms/step - loss: 4035.2239 - val_loss: 4204.2051
Epoch 8/50
23/23 - 0s - 4ms/step - loss: 3027.6152 - val_loss: 3519.0332
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 2618.8931 - val_loss: 3207.7017
Epoch 10/50
23/23 - 0s - 4ms/step - loss: 2434.0427 - val_loss: 3006.7778
Epoch 11/50
23/23 - 0s - 4ms/step - loss: 2303.8274 - val_loss: 2857.5881
Epoch 12/50
23/23 - 0s - 4ms/step - loss: 2196.7461 - val_loss: 2721.5278
Epoch 13/50
23/23 - 0s - 4ms/step - loss: 2097.9919 - val_loss: 2588.9170
Epoch 14/50
23/23 - 0s - 4ms/step - loss: 2004

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 45ms/step - loss: 4716.7134 - val_loss: 3280.9385
Epoch 2/50
23/23 - 0s - 4ms/step - loss: 2888.2607 - val_loss: 2338.4097
Epoch 3/50
23/23 - 0s - 4ms/step - loss: 2132.2156 - val_loss: 1799.3541
Epoch 4/50
23/23 - 0s - 4ms/step - loss: 1686.1873 - val_loss: 1440.6819
Epoch 5/50
23/23 - 0s - 4ms/step - loss: 1353.5358 - val_loss: 1166.3856
Epoch 6/50
23/23 - 0s - 4ms/step - loss: 1109.2115 - val_loss: 954.2648
Epoch 7/50
23/23 - 0s - 4ms/step - loss: 910.6373 - val_loss: 790.8459
Epoch 8/50
23/23 - 0s - 4ms/step - loss: 756.8855 - val_loss: 660.6411
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 633.4490 - val_loss: 556.8186
Epoch 10/50
23/23 - 0s - 5ms/step - loss: 537.5305 - val_loss: 481.2145
Epoch 11/50
23/23 - 0s - 5ms/step - loss: 453.1895 - val_loss: 415.1826
Epoch 12/50
23/23 - 0s - 4ms/step - loss: 395.5674 - val_loss: 371.8141
Epoch 13/50
23/23 - 0s - 4ms/step - loss: 341.6963 - val_loss: 330.8270
Epoch 14/50
23/23 - 0s - 4ms/step - loss: 302.4456 - val_loss: 306.1232


C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 44ms/step - loss: 53363.7109 - val_loss: 35081.5352
Epoch 2/50
23/23 - 0s - 5ms/step - loss: 23063.1738 - val_loss: 12114.0840
Epoch 3/50
23/23 - 0s - 5ms/step - loss: 6739.6416 - val_loss: 2975.6421
Epoch 4/50
23/23 - 0s - 5ms/step - loss: 1791.6614 - val_loss: 1247.1962
Epoch 5/50
23/23 - 0s - 5ms/step - loss: 1116.8687 - val_loss: 1105.6918
Epoch 6/50
23/23 - 0s - 5ms/step - loss: 1038.9487 - val_loss: 1036.0985
Epoch 7/50
23/23 - 0s - 5ms/step - loss: 969.7139 - val_loss: 973.8419
Epoch 8/50
23/23 - 0s - 4ms/step - loss: 903.4858 - val_loss: 909.2128
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 844.7876 - val_loss: 852.3412
Epoch 10/50
23/23 - 0s - 5ms/step - loss: 789.2620 - val_loss: 799.7230
Epoch 11/50
23/23 - 0s - 5ms/step - loss: 740.2834 - val_loss: 751.7842
Epoch 12/50
23/23 - 0s - 5ms/step - loss: 695.6665 - val_loss: 707.3674
Epoch 13/50
23/23 - 0s - 4ms/step - loss: 655.3151 - val_loss: 669.0386
Epoch 14/50
23/23 - 0s - 5ms/step - loss: 620.0041 - val_loss: 630.

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 46ms/step - loss: 7219.9741 - val_loss: 4827.9497
Epoch 2/50
23/23 - 0s - 5ms/step - loss: 3839.4041 - val_loss: 3756.0405
Epoch 3/50
23/23 - 0s - 5ms/step - loss: 3122.1724 - val_loss: 2928.6780
Epoch 4/50
23/23 - 0s - 5ms/step - loss: 2503.4983 - val_loss: 2341.1787
Epoch 5/50
23/23 - 0s - 5ms/step - loss: 2006.4607 - val_loss: 1883.8411
Epoch 6/50
23/23 - 0s - 4ms/step - loss: 1629.2458 - val_loss: 1548.3752
Epoch 7/50
23/23 - 0s - 5ms/step - loss: 1334.0618 - val_loss: 1282.1915
Epoch 8/50
23/23 - 0s - 5ms/step - loss: 1099.8348 - val_loss: 1068.8800
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 955.5541 - val_loss: 940.4185
Epoch 10/50
23/23 - 0s - 5ms/step - loss: 816.7198 - val_loss: 822.4124
Epoch 11/50
23/23 - 0s - 4ms/step - loss: 735.6741 - val_loss: 741.8845
Epoch 12/50
23/23 - 0s - 4ms/step - loss: 663.8641 - val_loss: 685.8208
Epoch 13/50
23/23 - 0s - 5ms/step - loss: 626.4972 - val_loss: 664.3900
Epoch 14/50
23/23 - 0s - 5ms/step - loss: 581.9590 - val_loss: 593.

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 48ms/step - loss: 378003.6250 - val_loss: 238563.8594
Epoch 2/50
23/23 - 0s - 4ms/step - loss: 161947.5938 - val_loss: 89315.6094
Epoch 3/50
23/23 - 0s - 5ms/step - loss: 56687.5547 - val_loss: 26536.9160
Epoch 4/50
23/23 - 0s - 5ms/step - loss: 16752.6875 - val_loss: 7431.5063
Epoch 5/50
23/23 - 0s - 4ms/step - loss: 6035.1870 - val_loss: 3521.8306
Epoch 6/50
23/23 - 0s - 4ms/step - loss: 4009.5767 - val_loss: 3044.5940
Epoch 7/50
23/23 - 0s - 4ms/step - loss: 3747.6558 - val_loss: 2969.8564
Epoch 8/50
23/23 - 0s - 4ms/step - loss: 3639.5759 - val_loss: 2891.9392
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 3542.0403 - val_loss: 2809.9900
Epoch 10/50
23/23 - 0s - 5ms/step - loss: 3435.7651 - val_loss: 2732.3777
Epoch 11/50
23/23 - 0s - 4ms/step - loss: 3333.9565 - val_loss: 2656.3308
Epoch 12/50
23/23 - 0s - 4ms/step - loss: 3230.8567 - val_loss: 2569.0264
Epoch 13/50
23/23 - 0s - 4ms/step - loss: 3124.1318 - val_loss: 2494.0723
Epoch 14/50
23/23 - 0s - 4ms/step - loss: 3022.

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 58ms/step - loss: 45998.1367 - val_loss: 32489.3828
Epoch 2/50
23/23 - 0s - 5ms/step - loss: 22016.2617 - val_loss: 12233.9160
Epoch 3/50
23/23 - 0s - 6ms/step - loss: 7340.2637 - val_loss: 3252.8213
Epoch 4/50
23/23 - 0s - 5ms/step - loss: 2626.9031 - val_loss: 1794.7762
Epoch 5/50
23/23 - 0s - 5ms/step - loss: 2138.5601 - val_loss: 1665.9867
Epoch 6/50
23/23 - 0s - 5ms/step - loss: 2003.9076 - val_loss: 1570.9449
Epoch 7/50
23/23 - 0s - 4ms/step - loss: 1884.3190 - val_loss: 1473.7931
Epoch 8/50
23/23 - 0s - 4ms/step - loss: 1771.9319 - val_loss: 1387.5331
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 1662.6635 - val_loss: 1295.8463
Epoch 10/50
23/23 - 0s - 4ms/step - loss: 1560.9556 - val_loss: 1215.5232
Epoch 11/50
23/23 - 0s - 4ms/step - loss: 1466.0872 - val_loss: 1141.3726
Epoch 12/50
23/23 - 0s - 4ms/step - loss: 1374.3396 - val_loss: 1073.2687
Epoch 13/50
23/23 - 0s - 4ms/step - loss: 1297.1520 - val_loss: 1008.4875
Epoch 14/50
23/23 - 0s - 4ms/step - loss: 1216.3611 -

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 66ms/step - loss: 65643.6953 - val_loss: 40831.0039
Epoch 2/50
23/23 - 0s - 6ms/step - loss: 27743.5820 - val_loss: 15502.5303
Epoch 3/50
23/23 - 0s - 6ms/step - loss: 9381.3984 - val_loss: 4583.2773
Epoch 4/50
23/23 - 0s - 6ms/step - loss: 2887.6499 - val_loss: 2465.6326
Epoch 5/50
23/23 - 0s - 5ms/step - loss: 1880.1176 - val_loss: 2033.5139
Epoch 6/50
23/23 - 0s - 5ms/step - loss: 1600.5288 - val_loss: 1753.9529
Epoch 7/50
23/23 - 0s - 5ms/step - loss: 1421.0020 - val_loss: 1584.8533
Epoch 8/50
23/23 - 0s - 6ms/step - loss: 1300.8634 - val_loss: 1451.8345
Epoch 9/50
23/23 - 0s - 7ms/step - loss: 1192.2754 - val_loss: 1331.9009
Epoch 10/50
23/23 - 0s - 6ms/step - loss: 1096.8323 - val_loss: 1223.4420
Epoch 11/50
23/23 - 0s - 5ms/step - loss: 1008.5644 - val_loss: 1123.7806
Epoch 12/50
23/23 - 0s - 5ms/step - loss: 924.8505 - val_loss: 1037.3912
Epoch 13/50
23/23 - 0s - 5ms/step - loss: 845.5126 - val_loss: 946.6747
Epoch 14/50
23/23 - 0s - 6ms/step - loss: 772.4146 - val

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 79ms/step - loss: 23214.3867 - val_loss: 5130.9531
Epoch 2/50
23/23 - 0s - 5ms/step - loss: 3584.7517 - val_loss: 3307.7405
Epoch 3/50
23/23 - 0s - 6ms/step - loss: 2892.8196 - val_loss: 2622.3696
Epoch 4/50
23/23 - 0s - 6ms/step - loss: 2503.0610 - val_loss: 2376.3513
Epoch 5/50
23/23 - 0s - 5ms/step - loss: 2248.2971 - val_loss: 2153.0120
Epoch 6/50
23/23 - 0s - 5ms/step - loss: 2045.8811 - val_loss: 1956.5634
Epoch 7/50
23/23 - 0s - 5ms/step - loss: 1858.6395 - val_loss: 1780.6329
Epoch 8/50
23/23 - 0s - 5ms/step - loss: 1687.6134 - val_loss: 1620.7714
Epoch 9/50
23/23 - 0s - 6ms/step - loss: 1540.0938 - val_loss: 1477.6182
Epoch 10/50
23/23 - 0s - 6ms/step - loss: 1407.8877 - val_loss: 1355.2651
Epoch 11/50
23/23 - 0s - 5ms/step - loss: 1292.4491 - val_loss: 1242.7987
Epoch 12/50
23/23 - 0s - 5ms/step - loss: 1188.0027 - val_loss: 1148.7753
Epoch 13/50
23/23 - 0s - 7ms/step - loss: 1100.9792 - val_loss: 1066.5732
Epoch 14/50
23/23 - 0s - 5ms/step - loss: 1028.4869 - va

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 49ms/step - loss: 197416.3438 - val_loss: 155949.7969
Epoch 2/50
23/23 - 0s - 5ms/step - loss: 122461.6250 - val_loss: 91888.0469
Epoch 3/50
23/23 - 0s - 5ms/step - loss: 66068.5469 - val_loss: 42127.7656
Epoch 4/50
23/23 - 0s - 4ms/step - loss: 25392.9766 - val_loss: 11809.7188
Epoch 5/50
23/23 - 0s - 4ms/step - loss: 6992.7310 - val_loss: 4017.2112
Epoch 6/50
23/23 - 0s - 4ms/step - loss: 4090.8967 - val_loss: 3557.8916
Epoch 7/50
23/23 - 0s - 5ms/step - loss: 3830.7690 - val_loss: 3355.9995
Epoch 8/50
23/23 - 0s - 4ms/step - loss: 3599.1846 - val_loss: 3166.4048
Epoch 9/50
23/23 - 0s - 5ms/step - loss: 3398.1145 - val_loss: 2982.1091
Epoch 10/50
23/23 - 0s - 5ms/step - loss: 3202.8108 - val_loss: 2811.0630
Epoch 11/50
23/23 - 0s - 4ms/step - loss: 3020.3459 - val_loss: 2646.1262
Epoch 12/50
23/23 - 0s - 5ms/step - loss: 2843.9248 - val_loss: 2491.2122
Epoch 13/50
23/23 - 0s - 5ms/step - loss: 2675.3979 - val_loss: 2345.5947
Epoch 14/50
23/23 - 0s - 4ms/step - loss: 2516

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 63ms/step - loss: 6067.7710 - val_loss: 5306.4023
Epoch 2/50
23/23 - 0s - 5ms/step - loss: 4065.0649 - val_loss: 3898.4700
Epoch 3/50
23/23 - 0s - 7ms/step - loss: 2973.0645 - val_loss: 2843.5200
Epoch 4/50
23/23 - 0s - 5ms/step - loss: 2258.7622 - val_loss: 2178.9900
Epoch 5/50
23/23 - 0s - 7ms/step - loss: 1753.3608 - val_loss: 1711.3119
Epoch 6/50
23/23 - 0s - 5ms/step - loss: 1388.0820 - val_loss: 1367.3859
Epoch 7/50
23/23 - 0s - 6ms/step - loss: 1106.8643 - val_loss: 1080.4692
Epoch 8/50
23/23 - 0s - 6ms/step - loss: 895.2737 - val_loss: 873.0788
Epoch 9/50
23/23 - 0s - 6ms/step - loss: 734.7875 - val_loss: 738.0804
Epoch 10/50
23/23 - 0s - 5ms/step - loss: 597.1763 - val_loss: 584.1005
Epoch 11/50
23/23 - 0s - 5ms/step - loss: 494.2497 - val_loss: 476.1877
Epoch 12/50
23/23 - 0s - 6ms/step - loss: 406.7023 - val_loss: 402.5238
Epoch 13/50
23/23 - 0s - 5ms/step - loss: 352.5000 - val_loss: 350.8377
Epoch 14/50
23/23 - 0s - 6ms/step - loss: 304.3201 - val_loss: 304.23

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 64ms/step - loss: 34589.1602 - val_loss: 22138.1523
Epoch 2/50
23/23 - 0s - 5ms/step - loss: 16298.0068 - val_loss: 11086.2598
Epoch 3/50
23/23 - 0s - 11ms/step - loss: 9843.7285 - val_loss: 8268.4561
Epoch 4/50
23/23 - 0s - 5ms/step - loss: 8216.5811 - val_loss: 7385.9888
Epoch 5/50
23/23 - 0s - 5ms/step - loss: 7388.9951 - val_loss: 6634.9468
Epoch 6/50
23/23 - 0s - 5ms/step - loss: 6644.8643 - val_loss: 5975.4541
Epoch 7/50
23/23 - 0s - 5ms/step - loss: 5963.6299 - val_loss: 5389.8975
Epoch 8/50
23/23 - 0s - 5ms/step - loss: 5376.1152 - val_loss: 4850.7817
Epoch 9/50
23/23 - 0s - 5ms/step - loss: 4826.8237 - val_loss: 4398.7642
Epoch 10/50
23/23 - 0s - 5ms/step - loss: 4390.6680 - val_loss: 3950.0122
Epoch 11/50
23/23 - 0s - 5ms/step - loss: 3945.3210 - val_loss: 3581.3037
Epoch 12/50
23/23 - 0s - 5ms/step - loss: 3580.4812 - val_loss: 3269.0911
Epoch 13/50
23/23 - 0s - 5ms/step - loss: 3270.5239 - val_loss: 2998.6741
Epoch 14/50
23/23 - 0s - 5ms/step - loss: 3006.8994 

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 63ms/step - loss: 669760.0625 - val_loss: 511582.2500
Epoch 2/50
23/23 - 0s - 5ms/step - loss: 405787.2812 - val_loss: 304144.4062
Epoch 3/50
23/23 - 0s - 5ms/step - loss: 238719.8906 - val_loss: 176721.2031
Epoch 4/50
23/23 - 0s - 5ms/step - loss: 137053.6094 - val_loss: 99526.3750
Epoch 5/50
23/23 - 0s - 7ms/step - loss: 75982.9141 - val_loss: 53951.3555
Epoch 6/50
23/23 - 0s - 5ms/step - loss: 40417.4688 - val_loss: 28071.8574
Epoch 7/50
23/23 - 0s - 5ms/step - loss: 20651.5098 - val_loss: 13985.7705
Epoch 8/50
23/23 - 0s - 5ms/step - loss: 10238.4844 - val_loss: 6901.7393
Epoch 9/50
23/23 - 0s - 5ms/step - loss: 5154.3794 - val_loss: 3623.8496
Epoch 10/50
23/23 - 0s - 7ms/step - loss: 2888.7651 - val_loss: 2167.4697
Epoch 11/50
23/23 - 0s - 5ms/step - loss: 1938.8750 - val_loss: 1569.8091
Epoch 12/50
23/23 - 0s - 5ms/step - loss: 1567.3135 - val_loss: 1350.0731
Epoch 13/50
23/23 - 0s - 7ms/step - loss: 1432.4187 - val_loss: 1263.5079
Epoch 14/50
23/23 - 0s - 5ms/step -

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 55ms/step - loss: 78484.8750 - val_loss: 25580.1660
Epoch 2/50
23/23 - 0s - 5ms/step - loss: 14435.2432 - val_loss: 8318.7266
Epoch 3/50
23/23 - 0s - 5ms/step - loss: 8008.2559 - val_loss: 7588.0776
Epoch 4/50
23/23 - 0s - 4ms/step - loss: 6961.2393 - val_loss: 6391.2744
Epoch 5/50
23/23 - 0s - 5ms/step - loss: 6059.0986 - val_loss: 5583.7896
Epoch 6/50
23/23 - 0s - 5ms/step - loss: 5299.1938 - val_loss: 4907.8018
Epoch 7/50
23/23 - 0s - 7ms/step - loss: 4609.0698 - val_loss: 4279.2578
Epoch 8/50
23/23 - 0s - 4ms/step - loss: 4053.3071 - val_loss: 3756.5381
Epoch 9/50
23/23 - 0s - 5ms/step - loss: 3556.4365 - val_loss: 3273.9023
Epoch 10/50
23/23 - 0s - 5ms/step - loss: 3129.9956 - val_loss: 2864.7732
Epoch 11/50
23/23 - 0s - 5ms/step - loss: 2743.9473 - val_loss: 2524.9119
Epoch 12/50
23/23 - 0s - 6ms/step - loss: 2412.3872 - val_loss: 2207.4805
Epoch 13/50
23/23 - 0s - 5ms/step - loss: 2117.2607 - val_loss: 1939.8226
Epoch 14/50
23/23 - 0s - 5ms/step - loss: 1870.7938 - 

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 53ms/step - loss: 26633.6445 - val_loss: 6181.1436
Epoch 2/50
23/23 - 0s - 5ms/step - loss: 4236.7983 - val_loss: 3372.1899
Epoch 3/50
23/23 - 0s - 5ms/step - loss: 3371.5876 - val_loss: 2796.4871
Epoch 4/50
23/23 - 0s - 5ms/step - loss: 2907.0728 - val_loss: 2500.2139
Epoch 5/50
23/23 - 0s - 5ms/step - loss: 2563.6501 - val_loss: 2174.8379
Epoch 6/50
23/23 - 0s - 4ms/step - loss: 2236.6038 - val_loss: 1897.8503
Epoch 7/50
23/23 - 0s - 4ms/step - loss: 1949.6956 - val_loss: 1645.9880
Epoch 8/50
23/23 - 0s - 5ms/step - loss: 1685.3235 - val_loss: 1430.2544
Epoch 9/50
23/23 - 0s - 5ms/step - loss: 1451.2404 - val_loss: 1223.9639
Epoch 10/50
23/23 - 0s - 5ms/step - loss: 1249.0499 - val_loss: 1053.5875
Epoch 11/50
23/23 - 0s - 5ms/step - loss: 1069.0139 - val_loss: 911.4485
Epoch 12/50
23/23 - 0s - 5ms/step - loss: 917.5397 - val_loss: 779.4161
Epoch 13/50
23/23 - 0s - 5ms/step - loss: 786.4437 - val_loss: 677.7456
Epoch 14/50
23/23 - 0s - 5ms/step - loss: 676.4487 - val_loss

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 46ms/step - loss: 475032.4688 - val_loss: 363423.5312
Epoch 2/50
23/23 - 0s - 5ms/step - loss: 299452.2812 - val_loss: 217886.1406
Epoch 3/50
23/23 - 0s - 4ms/step - loss: 174588.0781 - val_loss: 119800.3594
Epoch 4/50
23/23 - 0s - 4ms/step - loss: 94071.6875 - val_loss: 60020.8984
Epoch 5/50
23/23 - 0s - 4ms/step - loss: 46420.4805 - val_loss: 27932.3242
Epoch 6/50
23/23 - 0s - 4ms/step - loss: 21959.8320 - val_loss: 12967.3115
Epoch 7/50
23/23 - 0s - 4ms/step - loss: 10876.5352 - val_loss: 7204.1489
Epoch 8/50
23/23 - 0s - 4ms/step - loss: 6800.5430 - val_loss: 5293.0947
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 5358.6016 - val_loss: 4820.7749
Epoch 10/50
23/23 - 0s - 5ms/step - loss: 4918.4663 - val_loss: 4655.6157
Epoch 11/50
23/23 - 0s - 4ms/step - loss: 4706.7188 - val_loss: 4535.4663
Epoch 12/50
23/23 - 0s - 4ms/step - loss: 4547.8237 - val_loss: 4402.1104
Epoch 13/50
23/23 - 0s - 4ms/step - loss: 4395.1392 - val_loss: 4264.9536
Epoch 14/50
23/23 - 0s - 4ms/step - lo

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 46ms/step - loss: 5755.6807 - val_loss: 4144.4663
Epoch 2/50
23/23 - 0s - 4ms/step - loss: 3428.5427 - val_loss: 2771.4580
Epoch 3/50
23/23 - 0s - 4ms/step - loss: 2501.6079 - val_loss: 1944.2137
Epoch 4/50
23/23 - 0s - 4ms/step - loss: 1817.6399 - val_loss: 1398.5726
Epoch 5/50
23/23 - 0s - 4ms/step - loss: 1287.3344 - val_loss: 967.5010
Epoch 6/50
23/23 - 0s - 4ms/step - loss: 849.8480 - val_loss: 700.9286
Epoch 7/50
23/23 - 0s - 4ms/step - loss: 601.1460 - val_loss: 525.0353
Epoch 8/50
23/23 - 0s - 4ms/step - loss: 451.7160 - val_loss: 424.1254
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 362.2268 - val_loss: 356.4604
Epoch 10/50
23/23 - 0s - 4ms/step - loss: 311.6234 - val_loss: 309.3408
Epoch 11/50
23/23 - 0s - 4ms/step - loss: 267.3013 - val_loss: 270.2065
Epoch 12/50
23/23 - 0s - 4ms/step - loss: 235.0481 - val_loss: 244.9373
Epoch 13/50
23/23 - 0s - 4ms/step - loss: 212.7411 - val_loss: 230.5669
Epoch 14/50
23/23 - 0s - 4ms/step - loss: 205.5535 - val_loss: 221.9316
Ep

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 45ms/step - loss: 85017.8672 - val_loss: 37975.5352
Epoch 2/50
23/23 - 0s - 4ms/step - loss: 18840.7344 - val_loss: 5918.5054
Epoch 3/50
23/23 - 0s - 4ms/step - loss: 3310.5576 - val_loss: 1953.7589
Epoch 4/50
23/23 - 0s - 4ms/step - loss: 2053.7803 - val_loss: 1877.5455
Epoch 5/50
23/23 - 0s - 4ms/step - loss: 1975.3754 - val_loss: 1793.9495
Epoch 6/50
23/23 - 0s - 4ms/step - loss: 1884.4377 - val_loss: 1719.8528
Epoch 7/50
23/23 - 0s - 4ms/step - loss: 1802.6359 - val_loss: 1653.9565
Epoch 8/50
23/23 - 0s - 4ms/step - loss: 1721.8771 - val_loss: 1590.9501
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 1646.1855 - val_loss: 1531.0798
Epoch 10/50
23/23 - 0s - 4ms/step - loss: 1574.8132 - val_loss: 1469.4384
Epoch 11/50
23/23 - 0s - 4ms/step - loss: 1500.6056 - val_loss: 1410.5345
Epoch 12/50
23/23 - 0s - 4ms/step - loss: 1435.3733 - val_loss: 1360.7424
Epoch 13/50
23/23 - 0s - 4ms/step - loss: 1372.9894 - val_loss: 1307.3672
Epoch 14/50
23/23 - 0s - 5ms/step - loss: 1312.9047 - 

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 44ms/step - loss: 9832.0752 - val_loss: 7175.8550
Epoch 2/50
23/23 - 0s - 4ms/step - loss: 6663.3579 - val_loss: 4805.1094
Epoch 3/50
23/23 - 0s - 4ms/step - loss: 4408.4736 - val_loss: 3261.5105
Epoch 4/50
23/23 - 0s - 4ms/step - loss: 2977.6318 - val_loss: 2329.2412
Epoch 5/50
23/23 - 0s - 4ms/step - loss: 2134.5579 - val_loss: 1758.2703
Epoch 6/50
23/23 - 0s - 4ms/step - loss: 1620.5787 - val_loss: 1423.5598
Epoch 7/50
23/23 - 0s - 4ms/step - loss: 1302.9469 - val_loss: 1201.1041
Epoch 8/50
23/23 - 0s - 4ms/step - loss: 1084.2098 - val_loss: 1074.2185
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 948.3317 - val_loss: 975.9292
Epoch 10/50
23/23 - 0s - 4ms/step - loss: 871.6017 - val_loss: 910.0083
Epoch 11/50
23/23 - 0s - 5ms/step - loss: 800.0541 - val_loss: 856.0177
Epoch 12/50
23/23 - 0s - 4ms/step - loss: 754.1413 - val_loss: 822.3495
Epoch 13/50
23/23 - 0s - 4ms/step - loss: 718.7212 - val_loss: 777.7653
Epoch 14/50
23/23 - 0s - 4ms/step - loss: 692.0120 - val_loss: 745.

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 45ms/step - loss: 1876.3064 - val_loss: 1605.4171
Epoch 2/50
23/23 - 0s - 4ms/step - loss: 1203.4048 - val_loss: 1114.1903
Epoch 3/50
23/23 - 0s - 4ms/step - loss: 917.8937 - val_loss: 868.0865
Epoch 4/50
23/23 - 0s - 4ms/step - loss: 712.4591 - val_loss: 651.2664
Epoch 5/50
23/23 - 0s - 4ms/step - loss: 535.9355 - val_loss: 489.2799
Epoch 6/50
23/23 - 0s - 4ms/step - loss: 410.1729 - val_loss: 366.8776
Epoch 7/50
23/23 - 0s - 4ms/step - loss: 308.4886 - val_loss: 274.5543
Epoch 8/50
23/23 - 0s - 4ms/step - loss: 239.8979 - val_loss: 212.7997
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 194.8258 - val_loss: 175.4263
Epoch 10/50
23/23 - 0s - 4ms/step - loss: 165.0457 - val_loss: 145.4220
Epoch 11/50
23/23 - 0s - 4ms/step - loss: 144.9270 - val_loss: 131.3338
Epoch 12/50
23/23 - 0s - 4ms/step - loss: 133.7679 - val_loss: 118.2461
Epoch 13/50
23/23 - 0s - 5ms/step - loss: 125.4598 - val_loss: 113.0094
Epoch 14/50
23/23 - 0s - 4ms/step - loss: 121.8278 - val_loss: 106.3337
Epoch 1

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 45ms/step - loss: 142396.3594 - val_loss: 104400.0938
Epoch 2/50
23/23 - 0s - 4ms/step - loss: 79457.5703 - val_loss: 55481.6328
Epoch 3/50
23/23 - 0s - 4ms/step - loss: 40904.4062 - val_loss: 27364.3398
Epoch 4/50
23/23 - 0s - 4ms/step - loss: 19793.4473 - val_loss: 12802.5459
Epoch 5/50
23/23 - 0s - 4ms/step - loss: 9344.4502 - val_loss: 6218.2803
Epoch 6/50
23/23 - 0s - 4ms/step - loss: 4889.8091 - val_loss: 3520.5513
Epoch 7/50
23/23 - 0s - 4ms/step - loss: 3176.1174 - val_loss: 2571.6226
Epoch 8/50
23/23 - 0s - 4ms/step - loss: 2594.1384 - val_loss: 2256.2620
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 2397.8975 - val_loss: 2135.8000
Epoch 10/50
23/23 - 0s - 4ms/step - loss: 2302.4956 - val_loss: 2064.5789
Epoch 11/50
23/23 - 0s - 4ms/step - loss: 2234.7493 - val_loss: 2002.3876
Epoch 12/50
23/23 - 0s - 4ms/step - loss: 2169.7336 - val_loss: 1939.9772
Epoch 13/50
23/23 - 0s - 4ms/step - loss: 2104.8945 - val_loss: 1879.9501
Epoch 14/50
23/23 - 0s - 4ms/step - loss: 2043.

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 79ms/step - loss: 141986.7969 - val_loss: 77451.3672
Epoch 2/50
23/23 - 0s - 5ms/step - loss: 48757.2812 - val_loss: 20066.3906
Epoch 3/50
23/23 - 0s - 4ms/step - loss: 12648.4355 - val_loss: 6975.0444
Epoch 4/50
23/23 - 0s - 4ms/step - loss: 6356.1538 - val_loss: 6310.5483
Epoch 5/50
23/23 - 0s - 8ms/step - loss: 5661.1982 - val_loss: 5727.8486
Epoch 6/50
23/23 - 0s - 5ms/step - loss: 5142.3857 - val_loss: 5163.1855
Epoch 7/50
23/23 - 0s - 5ms/step - loss: 4676.9863 - val_loss: 4674.0107
Epoch 8/50
23/23 - 0s - 5ms/step - loss: 4255.5532 - val_loss: 4230.6221
Epoch 9/50
23/23 - 0s - 6ms/step - loss: 3864.0693 - val_loss: 3854.9263
Epoch 10/50
23/23 - 0s - 6ms/step - loss: 3507.1980 - val_loss: 3492.4736
Epoch 11/50
23/23 - 0s - 6ms/step - loss: 3189.1731 - val_loss: 3171.2969
Epoch 12/50
23/23 - 0s - 6ms/step - loss: 2900.3018 - val_loss: 2893.2566
Epoch 13/50
23/23 - 0s - 5ms/step - loss: 2648.6143 - val_loss: 2636.6211
Epoch 14/50
23/23 - 0s - 6ms/step - loss: 2414.7341

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 46ms/step - loss: 3433.3896 - val_loss: 3991.9155
Epoch 2/50
23/23 - 0s - 4ms/step - loss: 3008.9717 - val_loss: 3388.6104
Epoch 3/50
23/23 - 0s - 4ms/step - loss: 2583.5469 - val_loss: 2919.5151
Epoch 4/50
23/23 - 0s - 4ms/step - loss: 2244.8293 - val_loss: 2561.7122
Epoch 5/50
23/23 - 0s - 4ms/step - loss: 1973.2286 - val_loss: 2264.9500
Epoch 6/50
23/23 - 0s - 4ms/step - loss: 1720.3217 - val_loss: 1972.1228
Epoch 7/50
23/23 - 0s - 4ms/step - loss: 1511.9929 - val_loss: 1729.8811
Epoch 8/50
23/23 - 0s - 4ms/step - loss: 1284.9030 - val_loss: 1361.8511
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 968.9449 - val_loss: 1024.9482
Epoch 10/50
23/23 - 0s - 4ms/step - loss: 729.2435 - val_loss: 784.1938
Epoch 11/50
23/23 - 0s - 4ms/step - loss: 585.0707 - val_loss: 625.1446
Epoch 12/50
23/23 - 0s - 4ms/step - loss: 462.9970 - val_loss: 506.2884
Epoch 13/50
23/23 - 0s - 4ms/step - loss: 384.4647 - val_loss: 427.9068
Epoch 14/50
23/23 - 0s - 4ms/step - loss: 328.1741 - val_loss: 384

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 49ms/step - loss: 609433.1875 - val_loss: 480355.8438
Epoch 2/50
23/23 - 0s - 4ms/step - loss: 395622.5625 - val_loss: 304072.8438
Epoch 3/50
23/23 - 0s - 4ms/step - loss: 245901.8281 - val_loss: 184872.7188
Epoch 4/50
23/23 - 0s - 5ms/step - loss: 146733.2344 - val_loss: 108263.3750
Epoch 5/50
23/23 - 0s - 5ms/step - loss: 84332.4141 - val_loss: 60901.8203
Epoch 6/50
23/23 - 0s - 5ms/step - loss: 46801.9297 - val_loss: 33594.6289
Epoch 7/50
23/23 - 0s - 4ms/step - loss: 25815.3906 - val_loss: 18550.4355
Epoch 8/50
23/23 - 0s - 4ms/step - loss: 14552.1260 - val_loss: 11019.9688
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 9058.0127 - val_loss: 7357.8906
Epoch 10/50
23/23 - 0s - 4ms/step - loss: 6488.0610 - val_loss: 5655.7251
Epoch 11/50
23/23 - 0s - 4ms/step - loss: 5327.3594 - val_loss: 4937.0732
Epoch 12/50
23/23 - 0s - 4ms/step - loss: 4799.2603 - val_loss: 4574.2778
Epoch 13/50
23/23 - 0s - 5ms/step - loss: 4538.5952 - val_loss: 4358.2578
Epoch 14/50
23/23 - 0s - 5ms/step

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 54ms/step - loss: 6454.2495 - val_loss: 2392.2766
Epoch 2/50
23/23 - 0s - 4ms/step - loss: 1243.2776 - val_loss: 778.5012
Epoch 3/50
23/23 - 0s - 4ms/step - loss: 710.8093 - val_loss: 566.5281
Epoch 4/50
23/23 - 0s - 7ms/step - loss: 524.0821 - val_loss: 445.1145
Epoch 5/50
23/23 - 0s - 5ms/step - loss: 431.6800 - val_loss: 373.0532
Epoch 6/50
23/23 - 0s - 4ms/step - loss: 362.5186 - val_loss: 316.1343
Epoch 7/50
23/23 - 0s - 4ms/step - loss: 309.8248 - val_loss: 278.2198
Epoch 8/50
23/23 - 0s - 4ms/step - loss: 279.0604 - val_loss: 248.5995
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 253.2885 - val_loss: 226.3009
Epoch 10/50
23/23 - 0s - 4ms/step - loss: 235.2387 - val_loss: 208.6062
Epoch 11/50
23/23 - 0s - 4ms/step - loss: 217.3173 - val_loss: 191.4750
Epoch 12/50
23/23 - 0s - 4ms/step - loss: 205.7853 - val_loss: 184.2672
Epoch 13/50
23/23 - 0s - 4ms/step - loss: 194.6309 - val_loss: 176.9688
Epoch 14/50
23/23 - 0s - 4ms/step - loss: 180.5208 - val_loss: 163.5614
Epoch 15

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 45ms/step - loss: 33400.6484 - val_loss: 9296.5303
Epoch 2/50
23/23 - 0s - 4ms/step - loss: 2963.9363 - val_loss: 685.0388
Epoch 3/50
23/23 - 0s - 4ms/step - loss: 673.6052 - val_loss: 685.2755
Epoch 4/50
23/23 - 0s - 4ms/step - loss: 577.9333 - val_loss: 618.5359
Epoch 5/50
23/23 - 0s - 4ms/step - loss: 535.7097 - val_loss: 589.0139
Epoch 6/50
23/23 - 0s - 4ms/step - loss: 503.4831 - val_loss: 545.9598
Epoch 7/50
23/23 - 0s - 4ms/step - loss: 473.3759 - val_loss: 510.4695
Epoch 8/50
23/23 - 0s - 4ms/step - loss: 444.6493 - val_loss: 479.9236
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 418.0424 - val_loss: 447.7793
Epoch 10/50
23/23 - 0s - 4ms/step - loss: 392.4987 - val_loss: 420.2931
Epoch 11/50
23/23 - 0s - 4ms/step - loss: 368.3515 - val_loss: 390.1715
Epoch 12/50
23/23 - 0s - 4ms/step - loss: 345.4267 - val_loss: 365.9510
Epoch 13/50
23/23 - 0s - 4ms/step - loss: 324.5139 - val_loss: 342.9392
Epoch 14/50
23/23 - 0s - 4ms/step - loss: 304.6855 - val_loss: 320.5256
Epoch 1

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 45ms/step - loss: 6102.5298 - val_loss: 4562.9385
Epoch 2/50
23/23 - 0s - 4ms/step - loss: 3708.8853 - val_loss: 3280.3765
Epoch 3/50
23/23 - 0s - 4ms/step - loss: 2721.9370 - val_loss: 2378.0173
Epoch 4/50
23/23 - 0s - 4ms/step - loss: 2010.4467 - val_loss: 1857.2273
Epoch 5/50
23/23 - 0s - 4ms/step - loss: 1586.8761 - val_loss: 1518.6412
Epoch 6/50
23/23 - 0s - 4ms/step - loss: 1311.1682 - val_loss: 1256.5303
Epoch 7/50
23/23 - 0s - 5ms/step - loss: 1086.9513 - val_loss: 1055.5366
Epoch 8/50
23/23 - 0s - 6ms/step - loss: 928.3866 - val_loss: 887.9960
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 810.0724 - val_loss: 777.3644
Epoch 10/50
23/23 - 0s - 4ms/step - loss: 726.8277 - val_loss: 725.0644
Epoch 11/50
23/23 - 0s - 5ms/step - loss: 652.7046 - val_loss: 636.5851
Epoch 12/50
23/23 - 0s - 5ms/step - loss: 585.7358 - val_loss: 577.0569
Epoch 13/50
23/23 - 0s - 5ms/step - loss: 539.4132 - val_loss: 525.9288
Epoch 14/50
23/23 - 0s - 5ms/step - loss: 498.4598 - val_loss: 485.24

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 64ms/step - loss: 296073.6562 - val_loss: 201201.9531
Epoch 2/50
23/23 - 0s - 5ms/step - loss: 136723.7656 - val_loss: 84125.2734
Epoch 3/50
23/23 - 0s - 7ms/step - loss: 53136.7539 - val_loss: 29996.4336
Epoch 4/50
23/23 - 0s - 6ms/step - loss: 17712.1641 - val_loss: 9721.0645
Epoch 5/50
23/23 - 0s - 5ms/step - loss: 5760.8857 - val_loss: 3865.2075
Epoch 6/50
23/23 - 0s - 5ms/step - loss: 2850.2078 - val_loss: 2599.4612
Epoch 7/50
23/23 - 0s - 5ms/step - loss: 2312.1899 - val_loss: 2382.8552
Epoch 8/50
23/23 - 0s - 5ms/step - loss: 2192.2825 - val_loss: 2292.4390
Epoch 9/50
23/23 - 0s - 5ms/step - loss: 2120.6108 - val_loss: 2212.4856
Epoch 10/50
23/23 - 0s - 5ms/step - loss: 2048.2935 - val_loss: 2135.9702
Epoch 11/50
23/23 - 0s - 6ms/step - loss: 1978.0562 - val_loss: 2067.5923
Epoch 12/50
23/23 - 0s - 6ms/step - loss: 1906.9015 - val_loss: 1988.6226
Epoch 13/50
23/23 - 0s - 5ms/step - loss: 1835.0414 - val_loss: 1916.5188
Epoch 14/50
23/23 - 0s - 5ms/step - loss: 1766.

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 45ms/step - loss: 16553.2891 - val_loss: 6667.5913
Epoch 2/50
23/23 - 0s - 5ms/step - loss: 4930.5449 - val_loss: 2039.1608
Epoch 3/50
23/23 - 0s - 4ms/step - loss: 2300.0549 - val_loss: 1704.0610
Epoch 4/50
23/23 - 0s - 4ms/step - loss: 2007.6311 - val_loss: 1611.8524
Epoch 5/50
23/23 - 0s - 4ms/step - loss: 1841.9926 - val_loss: 1476.1902
Epoch 6/50
23/23 - 0s - 4ms/step - loss: 1681.4830 - val_loss: 1322.4717
Epoch 7/50
23/23 - 0s - 4ms/step - loss: 1532.5529 - val_loss: 1205.5010
Epoch 8/50
23/23 - 0s - 4ms/step - loss: 1391.1018 - val_loss: 1099.6769
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 1259.0026 - val_loss: 995.2554
Epoch 10/50
23/23 - 0s - 4ms/step - loss: 1137.0707 - val_loss: 899.6310
Epoch 11/50
23/23 - 0s - 4ms/step - loss: 1026.7415 - val_loss: 807.3138
Epoch 12/50
23/23 - 0s - 4ms/step - loss: 928.0023 - val_loss: 733.8469
Epoch 13/50
23/23 - 0s - 8ms/step - loss: 837.5760 - val_loss: 660.8560
Epoch 14/50
23/23 - 0s - 5ms/step - loss: 757.1716 - val_loss: 

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 46ms/step - loss: 7593.7925 - val_loss: 3179.4529
Epoch 2/50
23/23 - 0s - 4ms/step - loss: 3433.5454 - val_loss: 2517.9404
Epoch 3/50
23/23 - 0s - 5ms/step - loss: 2657.9409 - val_loss: 2171.4519
Epoch 4/50
23/23 - 0s - 4ms/step - loss: 2292.8298 - val_loss: 1845.9893
Epoch 5/50
23/23 - 0s - 4ms/step - loss: 1960.5161 - val_loss: 1572.2944
Epoch 6/50
23/23 - 0s - 4ms/step - loss: 1671.9736 - val_loss: 1337.9266
Epoch 7/50
23/23 - 0s - 4ms/step - loss: 1433.6074 - val_loss: 1134.3123
Epoch 8/50
23/23 - 0s - 4ms/step - loss: 1217.2681 - val_loss: 957.6870
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 1043.7957 - val_loss: 811.3013
Epoch 10/50
23/23 - 0s - 4ms/step - loss: 876.3513 - val_loss: 689.1657
Epoch 11/50
23/23 - 0s - 5ms/step - loss: 750.4255 - val_loss: 591.2652
Epoch 12/50
23/23 - 0s - 4ms/step - loss: 647.9937 - val_loss: 505.8885
Epoch 13/50
23/23 - 0s - 4ms/step - loss: 558.8535 - val_loss: 434.3880
Epoch 14/50
23/23 - 0s - 4ms/step - loss: 488.1296 - val_loss: 378.

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 45ms/step - loss: 55125.1445 - val_loss: 41875.0977
Epoch 2/50
23/23 - 0s - 4ms/step - loss: 32247.8340 - val_loss: 24394.1152
Epoch 3/50
23/23 - 0s - 4ms/step - loss: 18380.2793 - val_loss: 13607.1533
Epoch 4/50
23/23 - 0s - 4ms/step - loss: 9957.9521 - val_loss: 7180.9126
Epoch 5/50
23/23 - 0s - 4ms/step - loss: 5062.5132 - val_loss: 3559.0911
Epoch 6/50
23/23 - 0s - 4ms/step - loss: 2451.7437 - val_loss: 1753.7223
Epoch 7/50
23/23 - 0s - 4ms/step - loss: 1255.0253 - val_loss: 1010.4258
Epoch 8/50
23/23 - 0s - 4ms/step - loss: 823.3659 - val_loss: 747.0092
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 692.3154 - val_loss: 666.4315
Epoch 10/50
23/23 - 0s - 4ms/step - loss: 649.7789 - val_loss: 637.9405
Epoch 11/50
23/23 - 0s - 4ms/step - loss: 629.8411 - val_loss: 617.0886
Epoch 12/50
23/23 - 0s - 4ms/step - loss: 613.0919 - val_loss: 598.6654
Epoch 13/50
23/23 - 0s - 4ms/step - loss: 596.6704 - val_loss: 580.4749
Epoch 14/50
23/23 - 0s - 4ms/step - loss: 580.3083 - val_loss: 

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 45ms/step - loss: 71432.6250 - val_loss: 42881.1992
Epoch 2/50
23/23 - 0s - 5ms/step - loss: 24506.9492 - val_loss: 10982.9297
Epoch 3/50
23/23 - 0s - 4ms/step - loss: 5135.7017 - val_loss: 2250.1970
Epoch 4/50
23/23 - 0s - 4ms/step - loss: 1705.1233 - val_loss: 1625.1056
Epoch 5/50
23/23 - 0s - 4ms/step - loss: 1587.3326 - val_loss: 1536.6548
Epoch 6/50
23/23 - 0s - 4ms/step - loss: 1479.9517 - val_loss: 1470.1482
Epoch 7/50
23/23 - 0s - 4ms/step - loss: 1402.8467 - val_loss: 1387.5282
Epoch 8/50
23/23 - 0s - 4ms/step - loss: 1327.6737 - val_loss: 1314.3971
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 1257.1250 - val_loss: 1244.1333
Epoch 10/50
23/23 - 0s - 4ms/step - loss: 1193.2947 - val_loss: 1182.1893
Epoch 11/50
23/23 - 0s - 4ms/step - loss: 1133.5845 - val_loss: 1120.8911
Epoch 12/50
23/23 - 0s - 4ms/step - loss: 1075.5712 - val_loss: 1065.4702
Epoch 13/50
23/23 - 0s - 4ms/step - loss: 1024.4060 - val_loss: 1014.0049
Epoch 14/50
23/23 - 0s - 4ms/step - loss: 973.9763 - 

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 45ms/step - loss: 128185.0078 - val_loss: 63772.3125
Epoch 2/50
23/23 - 0s - 4ms/step - loss: 33829.7188 - val_loss: 12419.0674
Epoch 3/50
23/23 - 0s - 4ms/step - loss: 6119.2109 - val_loss: 2621.5803
Epoch 4/50
23/23 - 0s - 4ms/step - loss: 2393.4771 - val_loss: 1960.5487
Epoch 5/50
23/23 - 0s - 4ms/step - loss: 2182.5464 - val_loss: 1865.9421
Epoch 6/50
23/23 - 0s - 4ms/step - loss: 2065.8555 - val_loss: 1780.1420
Epoch 7/50
23/23 - 0s - 4ms/step - loss: 1954.7942 - val_loss: 1680.7749
Epoch 8/50
23/23 - 0s - 4ms/step - loss: 1844.7627 - val_loss: 1587.0439
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 1737.9579 - val_loss: 1493.4169
Epoch 10/50
23/23 - 0s - 5ms/step - loss: 1637.9364 - val_loss: 1405.4120
Epoch 11/50
23/23 - 0s - 4ms/step - loss: 1539.0076 - val_loss: 1326.6292
Epoch 12/50
23/23 - 0s - 4ms/step - loss: 1443.5225 - val_loss: 1242.4254
Epoch 13/50
23/23 - 0s - 4ms/step - loss: 1356.8682 - val_loss: 1169.8795
Epoch 14/50
23/23 - 0s - 4ms/step - loss: 1281.0177 

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 45ms/step - loss: 37862.0234 - val_loss: 9867.1348
Epoch 2/50
23/23 - 0s - 4ms/step - loss: 6678.8203 - val_loss: 3779.1917
Epoch 3/50
23/23 - 0s - 4ms/step - loss: 4258.6406 - val_loss: 3732.4021
Epoch 4/50
23/23 - 0s - 4ms/step - loss: 3923.6660 - val_loss: 3234.4924
Epoch 5/50
23/23 - 0s - 4ms/step - loss: 3560.9854 - val_loss: 2933.9653
Epoch 6/50
23/23 - 0s - 4ms/step - loss: 3229.8628 - val_loss: 2685.9966
Epoch 7/50
23/23 - 0s - 4ms/step - loss: 2926.0378 - val_loss: 2446.9690
Epoch 8/50
23/23 - 0s - 4ms/step - loss: 2631.9915 - val_loss: 2195.1755
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 2363.8784 - val_loss: 2003.5043
Epoch 10/50
23/23 - 0s - 4ms/step - loss: 2129.6619 - val_loss: 1795.0083
Epoch 11/50
23/23 - 0s - 4ms/step - loss: 1904.9097 - val_loss: 1622.2839
Epoch 12/50
23/23 - 0s - 4ms/step - loss: 1702.9939 - val_loss: 1450.6887
Epoch 13/50
23/23 - 0s - 4ms/step - loss: 1514.0475 - val_loss: 1327.6687
Epoch 14/50
23/23 - 0s - 4ms/step - loss: 1366.2385 - va

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 44ms/step - loss: 299302.4375 - val_loss: 210251.3125
Epoch 2/50
23/23 - 0s - 4ms/step - loss: 146457.0469 - val_loss: 89126.9922
Epoch 3/50
23/23 - 0s - 4ms/step - loss: 53443.1250 - val_loss: 26741.8711
Epoch 4/50
23/23 - 0s - 4ms/step - loss: 15064.1182 - val_loss: 9901.2363
Epoch 5/50
23/23 - 0s - 4ms/step - loss: 7714.2793 - val_loss: 8143.4814
Epoch 6/50
23/23 - 0s - 4ms/step - loss: 6948.1489 - val_loss: 7566.0493
Epoch 7/50
23/23 - 0s - 5ms/step - loss: 6445.8154 - val_loss: 7027.6523
Epoch 8/50
23/23 - 0s - 5ms/step - loss: 5981.1958 - val_loss: 6523.0283
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 5559.1641 - val_loss: 6058.4268
Epoch 10/50
23/23 - 0s - 4ms/step - loss: 5176.1323 - val_loss: 5652.9111
Epoch 11/50
23/23 - 0s - 4ms/step - loss: 4834.3867 - val_loss: 5267.9585
Epoch 12/50
23/23 - 0s - 4ms/step - loss: 4517.5269 - val_loss: 4925.0923
Epoch 13/50
23/23 - 0s - 4ms/step - loss: 4235.3213 - val_loss: 4601.2397
Epoch 14/50
23/23 - 0s - 4ms/step - loss: 3950.

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 46ms/step - loss: 1370371.5000 - val_loss: 1129913.1250
Epoch 2/50
23/23 - 0s - 4ms/step - loss: 969134.6250 - val_loss: 793035.3125
Epoch 3/50
23/23 - 0s - 4ms/step - loss: 674810.6250 - val_loss: 546352.2500
Epoch 4/50
23/23 - 0s - 4ms/step - loss: 458925.2812 - val_loss: 365998.3438
Epoch 5/50
23/23 - 0s - 4ms/step - loss: 301812.3125 - val_loss: 235672.1875
Epoch 6/50
23/23 - 0s - 4ms/step - loss: 189983.4375 - val_loss: 144252.7812
Epoch 7/50
23/23 - 0s - 4ms/step - loss: 113204.6797 - val_loss: 82994.9141
Epoch 8/50
23/23 - 0s - 4ms/step - loss: 63225.6211 - val_loss: 44784.8047
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 32999.6719 - val_loss: 22598.1699
Epoch 10/50
23/23 - 0s - 4ms/step - loss: 16165.6143 - val_loss: 10824.4365
Epoch 11/50
23/23 - 0s - 4ms/step - loss: 7675.5029 - val_loss: 5149.1099
Epoch 12/50
23/23 - 0s - 4ms/step - loss: 3775.4688 - val_loss: 2708.0100
Epoch 13/50
23/23 - 0s - 4ms/step - loss: 2175.5547 - val_loss: 1720.5425
Epoch 14/50
23/23 - 0s

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 45ms/step - loss: 37671.5820 - val_loss: 25415.7480
Epoch 2/50
23/23 - 0s - 4ms/step - loss: 17593.3848 - val_loss: 10484.5518
Epoch 3/50
23/23 - 0s - 4ms/step - loss: 6201.9854 - val_loss: 2734.9861
Epoch 4/50
23/23 - 0s - 4ms/step - loss: 1457.1542 - val_loss: 876.1573
Epoch 5/50
23/23 - 0s - 4ms/step - loss: 804.3455 - val_loss: 802.0134
Epoch 6/50
23/23 - 0s - 4ms/step - loss: 756.4666 - val_loss: 750.1788
Epoch 7/50
23/23 - 0s - 4ms/step - loss: 710.9929 - val_loss: 708.9837
Epoch 8/50
23/23 - 0s - 4ms/step - loss: 672.8881 - val_loss: 670.5923
Epoch 9/50
23/23 - 0s - 5ms/step - loss: 637.9713 - val_loss: 634.0568
Epoch 10/50
23/23 - 0s - 4ms/step - loss: 605.6622 - val_loss: 604.2808
Epoch 11/50
23/23 - 0s - 5ms/step - loss: 576.6572 - val_loss: 574.8984
Epoch 12/50
23/23 - 0s - 4ms/step - loss: 550.6478 - val_loss: 549.3375
Epoch 13/50
23/23 - 0s - 5ms/step - loss: 527.2036 - val_loss: 525.5414
Epoch 14/50
23/23 - 0s - 5ms/step - loss: 505.6223 - val_loss: 504.7172


C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 1s - 45ms/step - loss: 41484.3008 - val_loss: 9008.6250
Epoch 2/50
23/23 - 0s - 4ms/step - loss: 6959.0547 - val_loss: 5720.4302
Epoch 3/50
23/23 - 0s - 4ms/step - loss: 5737.5210 - val_loss: 4702.6733
Epoch 4/50
23/23 - 0s - 4ms/step - loss: 4977.5151 - val_loss: 4073.6748
Epoch 5/50
23/23 - 0s - 4ms/step - loss: 4475.0215 - val_loss: 3666.0396
Epoch 6/50
23/23 - 0s - 4ms/step - loss: 3986.4275 - val_loss: 3194.3083
Epoch 7/50
23/23 - 0s - 4ms/step - loss: 3521.5261 - val_loss: 2814.5686
Epoch 8/50
23/23 - 0s - 4ms/step - loss: 3099.2202 - val_loss: 2421.8772
Epoch 9/50
23/23 - 0s - 4ms/step - loss: 2707.4785 - val_loss: 2109.3518
Epoch 10/50
23/23 - 0s - 4ms/step - loss: 2348.4231 - val_loss: 1790.0215
Epoch 11/50
23/23 - 0s - 4ms/step - loss: 2029.1678 - val_loss: 1570.1111
Epoch 12/50
23/23 - 0s - 4ms/step - loss: 1755.7086 - val_loss: 1309.5793
Epoch 13/50
23/23 - 0s - 5ms/step - loss: 1508.0938 - val_loss: 1125.7102
Epoch 14/50
23/23 - 0s - 4ms/step - loss: 1296.8453 - va

In [21]:
# Compute the mean and standard deviation of the mean squared errors
import numpy as np
mean_mse = np.mean(mean_squared_errors)
std_mse = np.std(mean_squared_errors)

print("Mean of Mean Squared Errors: {}".format(mean_mse))
print("Standard Deviation of Mean Squared Errors: {}".format(std_mse))

Mean of Mean Squared Errors: 306.5576972136569
Standard Deviation of Mean Squared Errors: 236.66280913556488
